core/store/models/job_spec.go

```diff
@@ -194,7 +194,7 @@ const (
 // to a parent JobID.
 type Initiator struct {
 	ID        uint32 `json:"id" gorm:"primary_key;auto_increment"`
-	JobSpecID *ID    `json:"jobSpecId" gorm:"index;type:varchar(36) REFERENCES job_specs(id)"`
+	JobSpecID *ID    `json:"jobSpecId"`
```

Comment:
Is the struct tag just not needed anymore because of your previous work on foreign keys?  Seemed unrelated to this PR

Reply:
These gorm tags are basically useless.

They are only required for automigrate, which we shouldn't be using, and it gets even worse because they get out of sync with the schema and are misleading.